core/market/decentralized/src/db/dao/offer.rs

```diff
@@ -48,6 +48,13 @@ impl<'c> OfferDao<'c> {
         .await
     }
 
+    pub async fn get_offers(&self) -> DbResult<Vec<Offer>> {
+        readonly_transaction(self.pool, move |conn| {
+            Ok(dsl::market_offer.load::<Offer>(conn)?)
+        })
+        .await
+    }
+
```

Comment:
To be compatible with spec we should return only our Offers. Here we return all exiting offers.

Reply:
nice catch!